src/main/resources/data/precludedmodules.json

```diff
@@ -0,0 +1,7 @@
+{
+  "CS1010X" : "CS1101S",
```

Comment:
I think there are still a lot of preclusions HAHA like cs1010s, cs1010j, etc.

Reply:
Updated accordingly!